src/shims/unix/fs.rs

```diff
@@ -297,6 +321,12 @@ impl FileDescriptor for DummyOutput {
     fn dup<'tcx>(&mut self) -> io::Result<Box<dyn FileDescriptor>> {
         Ok(Box::new(DummyOutput))
     }
+
+    #[cfg(unix)]
+    fn as_unix_host_fd(&self) -> Option<i32> {
+        // Is there a host fd?
```

Comment:
This one can return None. If the atty stuff becomes important for output hiding, we can give this another go